pallets/did/src/tests.rs

```diff
@@ -453,12 +467,11 @@ fn check_successful_keys_deletion_update() {
 	old_did_details.update_delegation_key(did::DidVerificationKey::from(del_key.public()), 0u64);
 
 	// Remove both attestation and delegation key
-	let mut operation = generate_base_did_update_operation(alice_did.clone());
-	operation.attestation_key_update = did::DidVerificationKeyUpdateAction::Delete;
-	operation.delegation_key_update = did::DidVerificationKeyUpdateAction::Delete;
-	operation.tx_counter = old_did_details.last_tx_counter + 1u64;
+	let mut details = generate_base_did_update_details();
+	details.attestation_key_update = did::DidVerificationKeyUpdateAction::Delete;
+	details.delegation_key_update = did::DidVerificationKeyUpdateAction::Delete;
 
-	let signature = auth_key.sign(operation.encode().as_ref());
+	let _signature = auth_key.sign(details.encode().as_ref());
```

Comment:
Do we still need to sign here? Can we not just remove this line?

Reply:
Fault of `clippy --fix` 😅  Good catch, thanks!

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/231/commits/677a827c2531efc1e80df819b201df8fca8d0334